### ISSUE TITLE
need for a log file.

### ISSUE THREAD

**Issue by buluofeng2** [Megvii-BaseDetection/DenseTeacher#9]

**Comment by ZRandomize** [comment]
you are welcome
[example_log.log](https://github.com/Megvii-BaseDetection/DenseTeacher/files/9307690/example_log.log)

**Comment by buluofeng2** [comment]
okay, thank you!

**Comment by buluofeng2** [comment]
I have try this way in another detection framework, and i finish the four part of your code.
1. the data augmentation (4 img per batch for supervised branch and 2 img per batch for both teacher and student branch)
2. EMA, which start from 3000 iteration, with momentum=0.9996
3. burnin stategy，which cost 5000 iterations。
4. distill_loss for unsupervised branch.
5. the lr is warmup in first 1000 iters by linear to 0.01,  and lr is down to 0.001 in last epoch(iter 216200).
But in my 10% data experiments, i can only get the 31% mAP, which way during iter 66000, and the mAP doesn't increase from iter 66000.
I wonder if there are any other key factors that might be causing the accuracy to fail to improve. Or can i get a complete log of the 10%-data experiment, so that i can find the problem by myself.

**Comment by ZRandomize** [comment]
you can check the full config at the line 374 in the logfile. 
seem like we used equal number of sup and unsup images for each epoch, is that the problem?
we use unsup weight 4 for logits and 1 for deltas, is that the same?

the full log and performance curve is here
[full_log.log](https://github.com/Megvii-BaseDetection/DenseTeacher/files/9347775/full_log.log)
![image](https://user-images.githubusercontent.com/49480962/184794814-13b5f7f0-17f4-44f3-ae14-359256f522f3.png)

BTW, I'm wondering the performance of your baseline

**Comment by buluofeng2** [comment]
Thanks you so much.
